### PR TITLE
Sources: Add External Link Icons

### PR DESCRIPTION
**Pull request by amymok**
Adding the external link icons to sources page

## Additions

- Added classes to external link

## Review

- @stephanieosan @huetingj @virginiacc 

## Screenshots

Desktop:
screen shot 2015-09-08 at 4 46 55 pm
screen shot 2015-09-08 at 4 47 07 pm
screen shot 2015-09-08 at 4 47 18 pm

Mobile:
screen shot 2015-09-08 at 4 58 19 pm
screen shot 2015-09-08 at 4 58 30 pm
screen shot 2015-09-08 at 4 58 41 pm
screen shot 2015-09-08 at 4 58 53 pm

## Checklist

* [X] Changes are limited to a single goal (no scope creep)
* [X] Code can be automatically merged (no conflicts)
* [X] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by huetingj** [comment]
Looks good! :+1:

File: src/process/sources/index.html [hunk 1]
```diff
@@ -34,9 +34,17 @@ <h2 class="content-l_col-1-4 content-l_col">
                         <div class="sources-step">
                             <h3>Assess your spending</h3>
                             <h4>&ldquo;About half of Americans don’t have a budget.&rdquo;</h4>
-                            <p>Center for Financial Services Innovation. (March 2015). <i>Understanding and Improving Consumer Financial Health in America</i>, p. 9.  Available at: <a href="http://www.cfsinnovation.com/Document-Library/Understanding-Consumer-Financial-Health">http://www.cfsinnovation.com/Document-Library/Understanding-Consumer-Financial-Health</a>.</p>
+                            <p>Center for Financial Services Innovation. (March 2015). <i>Understanding and Improving Consumer Financial Health in America</i>, p. 9.  Available at: 
+                                <a class="icon-link icon-link__external-link" href="http://www.cfsinnovation.com/Document-Library/Understanding-Consumer-Financial-Health">
```
Comment (virginiacc) [inline src/process/sources/index.html]: Do we want to add the `target="_blank"` attribute so these  links open in a separate tab/window?

**Comment by virginiacc** [comment]
@amymok Looks great! :+1: 

 I'm wondering if we should open the links in a new tab/window since that's how we usually handle external links, but otherwise this is ready to go!

**Comment by mthibos** [comment]
@virginiacc @amymok , yes, definitely, we should open in new tabs

**Comment by virginiacc** [comment]
@mthibos @amymok 

@Scotchester pointed out that our standard approach to external links is to open them in the same window. I'm going to go ahead and merge this, and we can discuss the new window issue later.

**Comment by mthibos** [comment]
Hmm, interesting.  That is definitely not OaH's standard approach to external links.  For now, I think we should stick with OaH's standard approach.  Later, like after launch in a couple of weeks later, we can talk about whether OaH needs to harmonize with CFPB-at-large on the issue of link opening behavior.

**Pull request merged**